### DISCUSSION TITLE
How to change developer name?

### DISCUSSION THREAD

**Discussion by XFox111** [microsoft/MicrosoftEdge-Extensions#201] [Q&A] [answered]
I want to change my publisher name that is displayed on extension store page.

![image](https://github.com/user-attachments/assets/21ad3513-22b2-4936-84a7-431283f0716c)

I've checked every setting I have in Partner Center, but didn't find anything. The closest thing I found is profile contact info in account settings. But it didn't affect my display name.

![image](https://github.com/user-attachments/assets/42a7e005-fd83-460b-aa05-3c90855b2d3d)

How do I change it?

**Comment by ManikanthMSFT** [comment]
Hi @XFox111, thank you for reaching out to us. We recommend tapping the "Update" option, after which you can modify the name as desired.

Please let us know if you need any further assistance!

<img width="155" alt="image" src="https://github.com/user-attachments/assets/80283c62-5f24-44ae-a513-52c8b1d0993a">

**Reply by XFox111** [reply]
Yeah, that would make sense, wouldn't it? But as I said in the OP, it didn't affect my display name on Edge Add-ons

![image](https://github.com/user-attachments/assets/127454a5-ac0d-476a-998b-93f75cbf62b1)

UPD: I have noticed the warning notice under Display name. But if it's true, it is unclear how should I proceed with re-submitting my add-ons. I did push a few updates recently, but it didn't change anything, so clearly, it's not how it is done

**Reply by ManikanthMSFT** [reply]
Hi @XFox111, thank you for getting back to us. We kindly request that you wait a little longer and then check if the display name has updated. If you continue to experience the issue, please let us know.

**Reply by ManikanthMSFT** [reply]
Hi @XFox111, could you please confirm if your concern was resolved by changing your developer's name?

Thank you!

**Reply by XFox111** [reply]
Hi @ManikanthMSFT. No, it wasn't. It still shows the old one

**Reply by ManikanthMSFT** [reply]
Hey @XFox111, was your issue resolved? If not, could you please provide us with your seller ID so that we can verify and assist you accordingly?

Thanks!

**Reply by XFox111** [reply]
Hi @ManikanthMSFT, yes, it is still not resolved. Here's my seller ID, as you asked: 53891410

**Reply by ManikanthMSFT** [reply]
Hi @XFox111, could you please try to send us the screen recording of you trying to update your publisher's name to our [email](ext_dev_support@microsoft.com) by mentioning #201 so that we can assist you further.

**Reply by XFox111** [reply]
Hi @ManikanthMSFT, I have sent the email to `ext_dev_support@microsoft.com` (I assume this address is correct, because the link you've sent me is broken). The video is a bit over 50 MB, so I attached it as a link to my OneDrive, I hope it's okay.

The subject is: `RE #201: How to change developer name?`

**Comment by XFox111** [comment, marked as answer]
So, I was in contact with @ManikanthMSFT over email. Eventually the issue was resolved.

For those who has the similar issue, you need to do the following:
1. Go to Account Settings -> Legal Info
2. Click "Update" over "Contact Info"
3. Update your Publisher name and click "Save"
4. (Important) re-submit all your extensions (either by releasing a new version, or updating package store metadata)

Once your submission is approved the extension should display your new name.

For some reason it wasn't working for me for a while (I've updated the name a long time ago, and released multiple versions since then, but the name remained the same). Now it works though

Again, thanks to @ManikanthMSFT for their help :)
